notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/NotificationHub.java

```diff
@@ -35,19 +35,19 @@
     private SharedPreferences mPreferences;
     private static final String IS_ENABLED_PREFERENCE_KEY = "isEnabled";
 
-    private InstallationAdapter.Listener mOnSavedInstallation;
+    private InstallationAdapter.SaveListener mOnSavedInstallation;
     private InstallationAdapter.ErrorListener mOnInstallationFailure;
 
     NotificationHub() {
         mVisitors = new ArrayList<>();
 
         mOnInstallationFailure = new InstallationAdapter.ErrorListener() {
             @Override
-            public void onInstallationSaveError(Exception e) {
+            public void onInstallationOperationError(Exception e) {
                 Log.e("ANH", "unable to save installation: " + e.toString());

```

Comment:
Should we update this comment so it's not specific to the "save" operation?

Reply:
I was thinking that even if we're recycling the `ErrorListener` type that when we do plumb through the delete operation code that it would be a separate callback being registered. It's a little easier for customers that way, instead of filtering exceptions. :)